en/console-and-shells/commands.rst

```diff
@@ -244,6 +244,14 @@ You may need to call other commands from your command. You can use
 .. versionadded:: 3.8.0
     ``executeCommand()`` was added.
 
+
+.. note::
+
+    When calling ``executeCommand`` in a loop, it is recommended to pass in the
```

Comment:
```suggestion
    When calling ``executeCommand()`` in a loop, it is recommended to pass in the
```
maybe to clarify the method

Reply:
Happy to make that change, but do you think I should also apply that to the corresponding 4.x change that was already merged?  This was basically a copy/paste of that one

Reply:
Yes, make the change in both, please.

Reply:
done, and here is corresponding 4.x: https://github.com/cakephp/docs/pull/7309